### PR TITLE
Week 1 and 2 changes

### PR DESCRIPTION
**Pull request by kassenq**
- Update the README.md file to remove default descriptions and add my own
- Add links to socials and professional experiences
- Add pastel color randomizer to allow user to change background color
- Add Navbar using Bootstrap with Projects and Involvement pages. "Fun Things" page is reserved for functionality to be added later
- Change HTML/CSS spacing according to style guide

Edit:
- Add function to fetch fact data from server

### CODE REVIEW

File: portfolio/src/main/webapp/involvement.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+    <script src="script.js"></script>
+  </head>
+
+  <body id="body">
+    <div id="content">
+      <nav class="navbar navbar-default">
+        <div class="container-fluid">
+          <div class="navbar-header">
+          <a class="navbar-brand" href="#">Kassen's Portfolio</a>
```
Comment (qihuang2) [inline portfolio/src/main/webapp/involvement.html:23]: nit: indent this

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,17 +2,51 @@
...
+        </ul>
+        </div>
+      </nav>
+
+      <h1>Hi, I'm Kassen!</h1>
+      <p id="demo">I'm a born-and-raised Coloradan and rising junior at Duke. Welcome to my portfolio.</p>
+      <img src="/images/profilephoto.jpeg" alt="Profile Photo">
+
+      <br><br>
+      </hr>
```
Comment (qihuang2) [inline portfolio/src/main/webapp/index.html:38]: should this be "\<hr>" instead? here and other usages

File: portfolio/src/main/webapp/projects.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+          <li><a href="/involvement.html">Involvement</a></li>
+          <li><a href="#">Fun Things</a></li>
+          </ul>
+        </div>
+      </nav>
+
+      <h2> projects & work experience </h2>
+      <br><br>
+
+    <div class="project-container">
```
Comment (qihuang2) [inline portfolio/src/main/webapp/projects.html:36]: nit: this section needs to be indented

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -2,17 +2,51 @@
...
+
+      <div>
+        <a class="fa fa-linkedin" href="https://www.linkedin.com/in/kassenqian"></a>
+        <a class="fa fa-github" href="https://www.github.com/kassenq"></a>
+      </div>
+
+      <br><br>
+      </hr>
+
+      <p>Don't like this background color? Click here to change it:
```
Comment (qihuang2) [inline portfolio/src/main/webapp/index.html:48]: close the \<p> here

File: portfolio/src/main/webapp/involvement.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+      <a href="https://www.dtech.duke.edu">Duke Technology Scholars</a><br>
+      <a href="https://www.rewritingthecode.org">Rewriting the Code</a><br>
+      <a href="https://www.dukewit.herokuapp.com">Duke Women in Technology</a><br>
+      <a href="https://www.wfco.org">The Women's Foundation of Colorado</a><br>
+      <a href="https://www.yalamerica.org">Young Asian Leaders of America</a><br>
+
+      <br><br></hr>
+
+      <p>Don't like this background color? Click here to change it:
```
Comment (qihuang2) [inline portfolio/src/main/webapp/involvement.html:45]: close \<p> here as well

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -1,9 +1,29 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
-}
-
-#greeting-container {
-  margin-top: 20px;
+@import url("https://fonts.googleapis.com/css?family=Nunito+Sans:400,700");
```
Comment (qihuang2) [inline portfolio/src/main/webapp/style.css:1]: are you using the 700 font right now? if not remove it

File: portfolio/src/main/webapp/projects.html [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+      <div class="project">
+        <img class="project-img" src="/images/wit.png" href="https://www.dukewit.herokuapp.com">
+        <br>
+        <p>Co-President & Webmaster @ Duke Women in Technology</p>
+      </div>
+    </div
+
+    <br><br></hr>
+
+      <p>Don't like this background color? Click here to change it:
```
Comment (qihuang2) [inline portfolio/src/main/webapp/projects.html:64]: close \<p>

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (kassenq) [reply portfolio/src/main/webapp/index.html]: Changed this and once they actually showed up, I didn't like how they looked so I took them out

**Pull request merged**